lib/package.json

```diff
@@ -49,11 +49,16 @@
     "grunt-contrib-qunit": "^4.0.0",
     "grunt-contrib-uglify": "^5.0.1",
     "grunt-run": "^0.8.1",
-    "@nevware21/grunt-ts-plugin": "^0.2.2",
-    "grunt-tslint": "^5.0.2",
-    "tslint": "^5.19.0",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-microsoft-contrib": "^6.2.0",
+    "@nevware21/grunt-ts-plugin": "^0.3.0",
+    "@nevware21/grunt-eslint-ts": "^0.1.0",
+    "@typescript-eslint/eslint-plugin": "^4.28.0",
+    "@typescript-eslint/parser": "^4.28.0",
+    "eslint": "^7.29.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^5.1.0",
+    "eslint-plugin-security": "^1.4.0",
```

Comment:
This change is about adding the usage of this plugin to comply with corp governance policy